### ISSUE TITLE
[BUG] Isolation does not work anymore with CB Response 7.1 or higher

### ISSUE THREAD

**Issue by PaulMiskinis** [carbonblack/cbapi-python#220]
**I am seeing this behaviour on: (please complete the following information):**
 - OS: Any
 - Carbon Black Product: CB Response >=7.1

**Describe the bug**
Isolate and unisolate functions do not work anymore on servers running CB Response 7.1 or higher. 

**Steps to Reproduce**
Attempt to isolate or unisolate any sensor, on a server running version 7.1. 

```
Python 3.6.9 (default, Nov  7 2019, 10:44:02) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from cbapi.response import *
>>> c = CbResponseAPI()
>>> sensor = c.select(Sensor).where("hostname:LESANDA-NB1").first()
>>> sensor.unisolate()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/paulmiskinis/.local/lib/python3.6/site-packages/cbapi/response/models.py", line 808, in unisolate
    self.save()
  File "/home/paulmiskinis/.local/lib/python3.6/site-packages/cbapi/models.py", line 464, in save
    self._update_object()
  File "/home/paulmiskinis/.local/lib/python3.6/site-packages/cbapi/response/models.py", line 840, in _update_object
    return super(Sensor, self)._update_object()
  File "/home/paulmiskinis/.local/lib/python3.6/site-packages/cbapi/models.py", line 416, in _update_object
    data=self._info)
  File "/home/paulmiskinis/.local/lib/python3.6/site-packages/cbapi/connection.py", line 295, in api_json_request
    result = self.session.http_request(method, uri, headers=headers, data=raw_data, **kwargs)
  File "/home/paulmiskinis/.local/lib/python3.6/site-packages/cbapi/connection.py", line 192, in http_request
    raise ServerError(error_code=r.status_code, message=r.text)
cbapi.errors.ServerError: Received error code 400 from API: <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<title>400 Bad Request</title>
<h1>Bad Request</h1>
<p>The browser (or proxy) sent a request that this server could not understand.</p>
```

**Additional context**
Worked flawlessly on previous versions of CB Response prior to 7.1. I've tried manual HTTP PUT (which used to work too) but those do not work anymore either. It seems that CB changed something under the hood.

**Comment by kebringer-cb** [comment]
Hi Paul, thanks for raising the bug.  It is related to a change in CB Response 7.1.0.  Release notes are available on User Exchange, the ID is CB-28683.  We will be working on an update to CBAPI and will update this bug when it is released.
https://community.carbonblack.com/t5/Documentation-Downloads/CB-Response-7-1-0-Server-Release-Notes/ta-p/88027

**Comment by kebringer-cb** [comment]
This has been fixed and is available on the master branch, and also version 1.6.2 of the python CBAPI.  Please let us know if there are further issues.
